accessibility/accessibility-checklist.md

```diff
@@ -1,94 +1,209 @@
+* [Tool: Landmarks extension for Chrome, Firefox and Opera](http://matatk.agrip.org.uk/landmarks/)
+* [Tool: HTML5 accessibility feature support for sections](http://www.html5accessibility.com/#sections)
+
+
+### Declare a language attribute
+
+Declaring a language attribute on the HTML element allows a screen reader to read out text with correct pronunciation. It also helps search engines return language-specific results. 
+
+If you include text on a page that uses a different language to the main document (e.g. your HTML lang is `en` and you are including a passage in `jp`), also identify that text with its own lang attribute. 
+
+Resources: 
+
+* [Article: Using the HTML lang attribute](https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/)
+* [Samurai errata GL4: Clarify natural-language usage](http://www.wcagsamurai.org/erratas/errata-listing/#GL4)
+
+
+### Use a logical source order
+
+Your page should still make sense when you look at it without CSS. This is a core [progressive enhancement technique](../practices/progressive-enhancement.md). 
+
+Don't surprise users by using CSS to make changes to apparent source order. 
+
+Resources:
+
+* [Article: Logical document structures](http://www.bettertesting.co.uk/content/?p=1619)
+* [Article: Source order matters](http://adrianroselli.com/2015/09/source-order-matters.html)
+* [Tool: Wave Extension](https://wave.webaim.org/extension/)
+
+
+### Use headings correctly
+
+The [HTML5 outline algorithm](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines) can't yet be relied on to convey information to users. Freely use HTML5 sectioning elements, but use headings to define the structure of your document. 
+
+Use an appropriate heading for each section. You can visually hide headings, but make sure they're still accessible to screen reader users. 
+
+Don't skip heading levels.
+
+Resources: 
+
+* [Article: Semantic Structure](https://webaim.org/techniques/semanticstructure/)
+* [Tool: HeadingsMap extension for Chrome](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en)
+* [Tool: HeadingsMap extension for Firefox](https://addons.mozilla.org/en-GB/firefox/addon/headingsmap/)
+
+
+### Use progressive enhancement
+
+Progressive enhancement means starting with a logical and semantic HTML document with all key functionality, then enhancing it with an aesthetic layer (CSS) and behavioural layer (JavaScript). It has benefits beyond accessibility but is a key starting point for creating accessible experiences. 
+
+Resources: 
+
+* [Playbook: Progressive enhancement](../practices/progressive-enhancement.md)
+
+
+## Media
+
+### Audio or video elements should be accessible
+
+This could involve adding captions to videos, providing transcripts of audio content, or including audio description on videos. 
```

Comment:
Might it be worth adding something like "you may have to work with a third party to generate captions for your videos."

Reply:
I mentioned this above to Charlie, that it's something that would need to come from content editors. I have ✨grand ambitions ✨for this checklist and I envisage that kind of thing coming later once we've gathered editorial to our collective warm accessibility loving bosom. 

Does that sound sensible?